src/comps/EachCardLanding.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react'
+import '../styles/utils/utils.css'
+import '../styles/card/verticalCard.css'
+import { LazyLoadImage } from 'react-lazy-load-image-component';
+import { useProducts } from '../helpers/context/products-context';
+
+function EachCard({ index }) {
+    
+    const { state } = useProducts();
+
+  const [showImgPlaceholder, setShowImgPlaceholder] = useState(true);
+  const [showImgErrorPlaceholder, setShowImgErrorPlaceholder] = useState(false);
+
+  return (
+    <div className="each-card-wrapper each-card-landing-wrapper">
+        <div className="each-card-top">
+        {
+            showImgPlaceholder && <div className="productImg-placeholder"></div>
+            }                    {
+            (!state.products[Number(index)].image1 || showImgErrorPlaceholder) && <div className="showErrorDiv">404   Image not found!</div>
```

Comment:
nice work here but is it needed, u are putting the images in the DB

Reply:
I'm showing few images from db on homepage by random indexes.